components/internal-services/internal_service_request_role.yaml

```diff
@@ -0,0 +1,34 @@
+# permissions for end users to view applications.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: internal-service-request-role
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: 'true'
```

Comment:
I would not aggregate to view since there are verbs - delete, create, update, patch.

Reply:
removed